Postgres/AddServer.swift

```diff
@@ -13,17 +13,29 @@ class AddServerViewController: NSViewController, MainWindowModelConsumer {
 	@objc dynamic var mainWindowModel: MainWindowModel!
 	@objc dynamic var name: String = "New Server"
 	@objc dynamic var port: UInt = 5432
+    @objc dynamic var binPath: String = ""
+    @objc dynamic var binPathEditable: Bool = false
 	@objc dynamic var varPath: String = ""
 	
 	var availableBinaries: [PostgresBinary] = []
+    var customBinPath: String = FileManager().homeDirectoryForCurrentUser.path
+    
 	@objc dynamic var versions: [String] = []
 	@objc dynamic var selectedVersionIdx: Int = 0
+    
+    
 		
 	
 	override func viewDidLoad() {
 		loadVersions()
         if availableBinaries.indices.contains(selectedVersionIdx) {
+            binPath = availableBinaries[selectedVersionIdx].binPath
+            binPathEditable = false
             varPath = FileManager().applicationSupportDirectoryPath().appending("/var-\(availableBinaries[selectedVersionIdx].version)")
+        } else {
+            binPath = customBinPath
```

Comment:
I don't think setting this to the home directory makes sense. I'd just leave it at the last value instead